src/Containers/RightSideBar/EditText.tsx

```diff
@@ -0,0 +1,253 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { Select } from '@Inputs/Select';
+import { Bold } from '@styled-icons/boxicons-regular/Bold';
+import { Italic } from '@styled-icons/boxicons-regular/Italic';
+import { Underline } from '@styled-icons/boxicons-regular/Underline';
+import { TextColor } from '@styled-icons/foundation/TextColor';
+import { AlignMiddle } from '@styled-icons/boxicons-regular/AlignMiddle';
+import { AlignLeft } from '@styled-icons/boxicons-regular/AlignLeft';
+import { AlignRight } from '@styled-icons/boxicons-regular/AlignRight';
+
+// API KEY: AIzaSyBW6afsWt6KM6yvzXgfJ9UmUGImHvedZbc
+// STYLED COMPONENTS
+const Wrapper = styled.div``;
+const StyledSelect = styled(Select)`
+    width: 143px;
+`;
+const FontStyledSelect = styled(Select)`
+    width: 143px;
+`;
+
+const Title = styled.h1`
+    font-size: 18px;
+    padding-right: 20px;
+    padding-left: 20px;
+    padding-bottom: 5px;
+    padding-top: 20px;
+    text-align: left;
+`;
+const Txt = styled.h1`
+    text-align: left;
+    font-size: 15px;
+`;
+const StyleTxt = styled.h1`
+    float: right;
+    line-height: 4px;
+    font-size: 15px;
+    padding-left: 8px;
+`;
+
+const TxtWrapper = styled.div`
+    text-align: center;
+`;
+const Container = styled.div`
+    width: 150px;
+    margin-left: 10px;
+    margin-right: 10px;
+    margin-top: 5px;
+    display: inline-block;
+`;
+const SmallContainer = styled.div`
+    text-align: center;
+    margin-left: 18px;
+    margin-right: 18px;
+    display: inline-block;
+`;
+const StyledContainer = styled.div`
+    margin-top: 30px;
+    display: inline-block;
+    text-align: center;
+`;
+const StyledContainerNested = styled.div`
+    text-align: center;
+    margin-left: 20px;
+    margin-right: 20px;
+`;
+
+const StyledBold = styled(Bold)`
```

Comment:
All of these are all the same. Look at some other examples of how they do it though using styled.svg